.taskcluster.yml

```diff
@@ -158,7 +158,7 @@ tasks:
                           'tasks_for == "cron"': low
                           'tasks_for == "github-push"|| isPullRequest': very-low
                           $default: lowest  # tasks_for == 'action'
-                  retries: 5
+                  retries: 0
```

Comment:
I _think_ this will only disable retries on the decision task. I don't think there's any way to set a default number of retries per task unfortunately :(.

Reply:
Yeah. This makes me wonder if `taskgraph.transforms.task` should support setting max retries...